functions/Test-DbaIdentityUsage.ps1

```diff
@@ -0,0 +1,182 @@
+
+.DESCRIPTION 
+IDENTITY seeds have max values based off of their data type.  This module will locate identity columns and report the seed usage.
+
+
+.PARAMETER SqlServer
+Allows you to specify a comma separated list of servers to query.
+
+.PARAMETER SqlCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
+$cred = Get-Credential, this pass this $cred to the param. 
+
+Windows Authentication will be used if DestinationSqlCredential is not specified. To connect as a different Windows user, run PowerShell as that user.	
+
+.PARAMETER Threshold
+Allows you to specify a minimum % of the seed range being utilized.  This can be used to ignore seeds that have only utilized a small fraction of the range.
+
+.NOTES 
+Author: Brandon Abshire, netnerds.net
+
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK 
+https://dbatools.io/Test-DbaIdentityUsage
+
+.EXAMPLE   
+Test-DbaIdentityUsage -SqlServer sql2008, sqlserver2012
+Check identity seeds for servers sql2008 and sqlserver2012.
+
+.EXAMPLE   
+Test-DbaIdentityUsage -SqlServer sql2008 -Database TestDB
+Check identity seeds on server sql2008 for only the TestDB database
+
+.EXAMPLE   
+Test-DbaIdentityUsage -SqlServer sql2008 -Database TestDB -Threshold 20
+Check identity seeds on server sql2008 for only the TestDB database, limiting results to 20% utilization of seed range or higher
+
+
+#>
+	[CmdletBinding()]
+	Param (
+		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
+		[Alias("ServerInstance", "SqlInstance", "SqlServers")]
+        [string[]]$SqlServer,
+		[System.Management.Automation.PSCredential]$SqlCredential,
+        [parameter(Position = 1, Mandatory = $false)]
+		[int]$Threshold,
+        [parameter(Position = 2, Mandatory = $false)]
+        [switch]$NoSystemDb
+	)
+	
+	DynamicParam {
+		if ($SqlServer) {
+			return Get-ParamSqlDatabases -SqlServer $SqlServer[0] -SqlCredential $Credential
+		}
+	}
+
+	BEGIN
+	{
+
+        $databases = $psboundparameters.Databases
+		$exclude = $psboundparameters.Exclude
+        
+        $threshold = 0
+        $threshold = $psboundparameters.Threshold
+
+        $collection = New-Object System.Collections.ArrayList
+
+        $sql = "Select DB_NAME(), Object_Name(id.object_id) As [table_name]
```

Comment:
This query does not take into account columns that starts with negative values.
I will give you a query that I use (on slack)

Output example:
![image](https://cloud.githubusercontent.com/assets/19521315/23372307/f9020eb2-fd12-11e6-845d-0080ee0b3ad8.png)